yuzu linux installer (#328)
yuzu-emu/yuzu-emu.github.io: site/content/entry/yuzu-linux-installer/index.md

+++

> date = "2022-08-10T23:00:00+05:30"

Reviewer: ```suggestion
date = "2022-08-11T23:00:00+05:30"
```

title = "New Feature Release - Installer for Linux"
author = "CaptV0rt3x"
forum = 612296
+++

> Hey there yuz-ers!

Reviewer: ```suggestion
Hey there, yuz-ers!
```

Guess what? We've got some news to share.

> We're excited to announce Linux support for our yuzu installer!

Reviewer: ```suggestion
We're excited to announce that the yuzu installer now supports Linux!
```

Let's jump right in!

<!--more-->

## What's happening?

> We are very excited to announce Linux compatibility for our yuzu installer.

Reviewer: ```suggestion
As we mentioned, the yuzu installer is available for Linux.
```

> The installer will distribute AppImages of mainline and early access builds for our Linux users.

Reviewer: ```suggestion
The installer will distribute `AppImages` of Mainline and Early Access builds for our Linux users.
```

> You can now download this installer from our [downloads page](https://yuzu-emu.org/downloads/).

Reviewer: ```suggestion
You can now download the new installer from our [Downloads page](https://yuzu-emu.org/downloads/).
```

> (Note: You need to be a yuzu Patreon Subscriber to get access to the early access builds)

Reviewer: ```suggestion
(Note: Early Access builds require a subscription to our [Patreon](https://www.patreon.com/yuzuteam). We appreciate any and all support!)
```

## Why?

> When we originally released our installer, our Patreon userbase was predominantly Windows users and truth-be-told, it still is.
> That's why the installer's Linux compatibility wasn't a priority for us then.

Reviewer: ```suggestion
When we originally released our installer, our user base was comprised predominantly of Windows users.
Although a weak excuse, that's why the installer's Linux compatibility wasn't a priority for us then.
```

> {{< imgs
> 	"./maintenance.png| As a current limitation, Patreon users will have to paste their token by pressing Ctrl + V"
>   >}}

Reviewer: ```suggestion
```
This image's description text doesn't match the content of the image, and the image itself doesn't add much to the article. Removing

> But, over the years we've seen consistent growth in the number of our Linux users 
> and realised that we had to ensure that they reaped the benefits of the installer too.

Reviewer: ```suggestion
But, over the years we've seen consistent growth in the number of our Linux users 
and realized that we had to ensure that they reaped the benefits of the installer too. We're so happy our Linux community continues to expand and support us.
```

{{< imgs
	"./install.png| It's never a bad idea to pick both builds"
  >}}

Thanks to the efforts of [toastUnlimited](https://github.com/lat9nq), who implemented Linux compatibility for the installer,

> we're now able to address this long due feature request.

Reviewer: ```suggestion
we're now able to address this long overdue feature request.
```

{{< imgs

> 	"./download.png| Remember we update daily!"

Reviewer: ```suggestion
	"./download.png| Remember, we update daily!"
```

  >}}

## Fin

> As with any new update, we've internally tested this to some extent. 

Reviewer: ```suggestion
As with any new update, we've tested the installer internally, but our efforts can't compare to the community as a whole. 
```

> However, it is very easy for us to have missed something or for you to encounter a wierd bug or issue.
> So, please reach out to us on our [Discord](https://discord.gg/u77vRWY) and report these bugs, so that we can quickly address them.

Reviewer: ```suggestion
We could have easily missed an edge case or some weird bug or issue.
So, please reach out to us on our [Discord](https://discord.gg/u77vRWY) and report any bugs you find, so that we can quickly address them.
```

> ### That's all we have for now, until next time! Happy emulating!

Reviewer: ```suggestion
That's all we have for now, until next time! Happy emulating!
```
